zilencer/management/commands/populate_db.py

```diff
@@ -177,14 +179,26 @@ def handle(self, **options):
                                  Recipient.objects.filter(type=Recipient.STREAM)]
 
         # Extract a list of all users
-        user_profiles = [user_profile.id for user_profile in UserProfile.objects.all()] # type: List[int]
+        user_profiles = list(UserProfile.objects.all()) # type: List[UserProfile]
+
+        if not options["test_suite"]:
+            # Populate users with some bar data
+            for user in user_profiles:
+                status = 1 # type: int
+                date = datetime.datetime.today() # type: datetime.datetime
+                client = get_client("website")
+                for i in range(3):
+                    client = get_client("API")
```

Comment:
You are overriding `client` here. `client` will never be equal to`get_client("website")` due to `client = get_client("API")`